cmd/dockerd/daemon.go

```diff
@@ -715,16 +715,13 @@ func buildRouters(opts routerOptions) []router.Router {
 		swarmrouter.NewRouter(opts.cluster),
 		pluginrouter.NewRouter(opts.daemon.PluginManager()),
 		distributionrouter.NewRouter(opts.daemon.ImageBackend()),
+		network.NewRouter(opts.daemon, opts.cluster),
```

Comment:
This line will conflict with https://github.com/moby/moby/pull/49051, so depending on which one gets merged first may require a rebase / conflict fix